examples/systemd/conf/agent/main.conf

```diff
@@ -0,0 +1,28 @@
+agent {
+    log_level = "DEBUG"
+    trust_domain = "example.org"
+    server_address = "localhost"
+    server_port = 8081
+
+    # Insecure bootstrap is NOT appropriate for production use but is ok for
+    # simple testing/evaluation purposes.
+    insecure_bootstrap = true
+
+    join_token = "cdf1885a-1db8-4a83-aa16-ad8c84761fa8"
```

Comment:
This is not the best, but I imagine you want to wait for something like that dns/http pop attestor being available, right?

Reply:
yeah, that would be a better default once its available.

This is just an example config file really. Its expected an end user will need to update it to use it properly.